### PR TITLE
Add cheatsheet to README

### LINKED ISSUE #203: Add cheatsheet to README

### PR DESCRIPTION
**Pull request by edelarua**
Closes #203

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
 
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-# rlistings <a href='https://github.com/insightsengineering/rlistings'><img src="man/figures/logo.png" align="right" height="200" width="200"/></a>
+# rlistings
 
 <!-- start badges -->
+
 [![Check
-🛠](https://github.com/insightsengineering/rlistings/actions/workflows/check.yaml/badge.svg)](https://insightsengineering.github.io/rlistings/main/unit-test-report/)
```
Comment (shajoezhu) [inline README.md:8]: as suggested here, i am a bit inclined to keep the unit-test-report https://github.com/insightsengineering/rlistings/pull/202#discussion_r1513736161

if this is misleading, perhaps could do, 

unittest-report -- with the link to the report
check --  with the workflow check link

Comment (shajoezhu) [reply README.md:8]: @Melkiades , any thoughts?

**Review by shajoezhu — APPROVED** [review]
lgtm, thanks @edelarua , with one minor comment with the check link

File: README.md (hunk 1 above)
Comment (edelarua) [reply README.md:8]: @shajoezhu I didn’t change anything in this part of the file so nothing should be missing unless the Rmd file was changed and not re-knit. I think the formatting just changed slightly when I knit the Rmd file, but I will double check and make changes if needed

Comment (shajoezhu) [reply README.md:8]: I see! It traces back to here. 

https://github.com/insightsengineering/rlistings/blame/d4e071dbb46f0652dbf425a599297d395d7e9ab8/README.Rmd#L19

@pawelru , should we keep using the unit-test-report link? or just switch it to the check history?

https://github.com/insightsengineering/rlistings/actions/workflows/check.yaml is not very useful, is it?

Comment (pawelru) [reply README.md:8]: Hmmm... I don't have a strong view on this to be honest. For now, for consistency, I would suggest to go with what's in other packages and we can open a cross-repo discussion what should be there.

**Pull request merged**